src/lib/components/FieldSelect.svelte

```diff
@@ -149,6 +204,18 @@
 	}
 
 	:global(.field-combobox-item-label) {
-		@apply w-full;
+		@apply grid w-full cursor-pointer grid-cols-[auto,max-content] gap-x-1;
+	}
+
+	:global(.field-combobox-item-label-option) {
+		@apply overflow-hidden text-ellipsis text-nowrap;
+	}
```

Comment:
Fixed the styling for mobile:

![image](https://github.com/user-attachments/assets/955e6ba8-2f4a-465d-a8b3-48c5123c3cdf)